### PR TITLE
second challenge update

### PR DESCRIPTION
**Pull request by modiboCamara**
Here is a proposed solution for the second challenge.

### CODE REVIEW

File: js/script.js
Comment (rajeevkumar504) [inline js/script.js:1]: Why do you like to use var instead of "let" and "const"?

Comment (rajeevkumar504) [reply js/script.js:1]: I've also learned it, thankyou @modiboCamara for resolving this issue.

**Review by rajeevkumar504 — APPROVED** [review]
I'm glad to collaborate with you. Thank you, @modiboCamara!

**Comment by modiboCamara** [comment]
Looking forward to further collobarations.

**Comment by modiboCamara** [comment]
the reason a used var is purely habitual, when i first got introduced to  javascript variables where declared using var but it is better to use const and let to declare you variable.

**Pull request merged**